beszel: add package for hub

hgy59 opened the pull request:
## Description

Add beszel-hub package

beszel-agent is already provided by #6940 (and fixed in #7343)


## Checklist

- [x] Build rule `all-supported` completed successfully
- [x] New installation of package completed successfully
- [ ] Package upgrade completed successfully (Manually install the package again)
- [ ] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Package-Update-Policy#tests-checks)
- [ ] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created

### Type of change

<!--Please use any relevant tags.-->
- [x] New Package

hgy59 committed:
beszel: add packages for hub and agent

hgy59 committed:
Merge branch 'master' of https://github.com/hgy59/spksrc into add_beszel

edersong commented:
It's working fine so far, thanks!!

virusek392programmer commented:
Hi, I have problem with Urbackup on Synology. "After launching the application, it runs for a moment (green text), and then it shuts down (gray text). This situation occurs on every Synology device when I start the Urbackup program.
![obraz](https://github.com/user-attachments/assets/02f5ad45-3803-4046-84b3-0745ab6b917f)
![obraz](https://github.com/user-attachments/assets/cc627d1d-0bfc-46c1-b7fa-95549c9c9c82)

hgy59 committed:
Merge branch 'master' of https://github.com/hgy59/spksrc into add_beszel

hgy59 committed:
Merge branch 'master' of https://github.com/hgy59/spksrc into add_beszel

hgy59 committed:
beszel: update to v0.18.7

- update hub and agent to v0.18.7

hgy59 committed:
beszel-agent: downgrade go to fix a runtime error on armv7

hgy59 committed:
Merge branch 'master' of https://github.com/hgy59/spksrc into add_beszel

hgy59 committed:
beszel-agent: update go to v1.26.3

- adjust patch to use go 1.26.3 to fix a regression on armv7

hgy59 committed:
Merge branch 'master' of https://github.com/hgy59/spksrc into add_beszel

hgy59 committed:
unify beszel-agent and beszel-hub

- adjustments for framework changes in #7280
- rename patch from upgrade to downgrade
- use identical patches for agent and hub
